content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -44,6 +44,8 @@ caption="GKE-IAM Role"
 1.  Once you have an account and project enabled, click the following link to open the Deployment Manager.
 
     [Istio GKE Deployment Manager](https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl={{< github_file >}}/install/gcp/deployment_manager/istio-cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin)
+    
+    Note: You can also run Deployment Manager with `gcloud` if you are familiar, [see documentation on github](https://github.com/istio/istio/tree/master/install/gcp/deployment_manager).
```

Comment:
Replace Note: with >

Suggest changing sentence to:

You may also perform this task [from the command line using `gcloud`]({{< github_tree >}}/install/gcp/deployment_manager)

Reply:
Good suggestion, I updated it 😄 👍